### ISSUE TITLE
raw post requests puts new lines

### ISSUE THREAD

**Issue by DeC1pher** [projectdiscovery/nuclei#103]
Hey Team,

at first, thanks for that **awesome** tool!

I have following issue.
When I create a **fuzzing** template with a raw POST request it adds lines which gives wrong http responses.

```
id: xy

info:
  name: TEST
  author:
  severity: critical

requests:
  - payloads:
      list_url: /usr/share/wordlists/xxe.txt
    attack: sniper # Available options: sniper, pitchfork and clusterbomb
    raw:
      - |
        POST {{list_url}} HTTP/1.1
        Host: {{Hostname}}
        User-Agent: curl
        Accept-Encoding: gzip, deflate
        Accept: */*
        Connection: close
        Referer: {{Hostname}}
        Cookie: true
        Content-Type: application/x-www-form-urlencoded

        state = xy

    matchers:
      - type: word
        words:
          - "xy"
```

Now with following call

`nuclei -l ../subd/test.txt -t ~/nuclei-templates/test.yaml -proxy-url http://localhost:8080`

I see following in the burp proxy:

```
1. POST /libs/test.js HTTP/1.1
2. Host: www.xy.com
3. User-Agent: curl
4. Content-Length: 14
5. Accept: */*
6. Accept-Encoding: gzip, deflate
7. Connection: close
8. Content-Type: application/x-www-form-urlencoded
9. Cookie: true
10. Referer: www.xy.com
11.
12.
13.
14. state = xy
15.
16.
```
Which gives me for my testing a http 500. 
> Looks like here are added unnesseray new lines.

If I now change it to for example following post request (which is normal header -> CRLF -> payload):

```
1. POST /libs/test.js HTTP/1.1
2. Host: www.xy.com
3. User-Agent: curl
4. Content-Length: 10
5. Accept: */*
6. Accept-Encoding: gzip, deflate
7. Connection: close
8. Content-Type: application/x-www-form-urlencoded
9. Cookie: true
10. Referer: www.xy.com
11.
12. state = xy
```
-> Note I removed 11,12,15,16 and its working.
So in this case its all fine. Please note the content-length.

It looks for me, like its a bug when fuzzing with POST requests, that these 4 new lines see above are added.

For example the template `cves/CVE-2020-12720.yaml` also uses raw post requests and here it looks good ` header -> CRLF -> payload` - no extra new lines.

Hope you can help me.

Best Regards

**Comment by bauthard** [comment]
Thank you @DeC1pher for the detailed issue, we will try to reproduce this and fix if it's a bug in fuzz feature.

**Comment by Ice3man543** [comment]
This has been fixed by PR #121. Thank you for the issue @DeC1pher.
